public/plugin.tsx

```diff
@@ -64,6 +77,82 @@ export class AlertingPlugin implements Plugin<AlertingSetup, AlertingStart, Aler
       },
     });
 
+    if (core.chrome.navGroup.getNavGroupEnabled()) {
+      // register applications with category and use case information
+      core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.observability, [
+        {
+          id: PLUGIN_NAME,
+          category: DEFAULT_APP_CATEGORIES.detect,
+          showInAllNavGroup: true
+        }
+      ])
+      core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS['security-analytics'], [
+        {
+          id: PLUGIN_NAME,
+          category: DEFAULT_APP_CATEGORIES.detect,
+          showInAllNavGroup: false
+        }
+      ])
+
+      // channels route
+      core.application.register({
```

Comment:
![image.png](https://github.com/user-attachments/assets/222d4e1a-3540-4f7f-808f-a5a4fe374be2)

Actually I am not very sure if we should register all the sub items of alerting plugin into left navigation. If it is aligned with requirement, then we are good.

Reply:
there are more changes coming as per new mocks